requirements-docs.in

```diff
@@ -0,0 +1,20 @@
+markdown-exec
```

Comment:
The dependencies needed to generate the docs will now follow the same pattern as prod and dev depencies - we define `requirements.in` file, and then use [pip-compile](https://pypi.org/project/pip-tools/) to generate the lock file, e.g.:

```sh
pip-compile requirements-docs.in -o requirements-docs.txt
```

Next step could then be to migrate to a package manager like uv / poetry / hatch, so that we don't have to manually modify the requirements files. If / when we do so, we should move all environments (prod, dev, docs), so that don't end up using mutiple ways for managing dependencies.